app/services/IdentityService.scala

```diff
@@ -0,0 +1,151 @@
+package services
+
+import com.amazonaws.regions.{Region, Regions}
+import com.gu.identity.play.IdUser
+import com.gu.monitoring.{AuthenticationMetrics, CloudWatch, RequestMetrics, StatusMetrics}
+import com.typesafe.scalalogging.LazyLogging
+import configuration.Config
+import model.PersonalData
+import play.api.Logger
+import play.api.Play.current
+import play.api.libs.json._
+import play.api.libs.ws.{WS, WSRequest, WSResponse}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+case class GuestUserNotCreated(s : String) extends RuntimeException
```

Comment:
Maybe doesn't need to extend `RuntimeException`, as it's never actually thrown as an exception?

There doesn't seem to be any handling of this error condition at the moment either... we should probably just throw an InternalServerError for the moment.